ui/src/shared/reducers/sources.js

```diff
@@ -1,4 +1,8 @@
-export default function sources(state = [], action) {
+const getInitialState = () => []
```

Comment:
I felt that way too at first, but I kind of like it. I guess it was intended to accomplish the same thing as the spread operator or Object.assign when that pattern was introduced, to create a new copy of initialState every invocation. That way we ensure we don't mutate initialState, but then again, we are spreading from initialState elsewhere. So yeah, let's choose one or the other. Thoughts?